### PR TITLE
Master rental search shda

### PR DESCRIPTION
**Pull request by donnie-blurry**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/website_sale/views/snippets/snippets.xml [hunk 1]
```diff
@@ -80,7 +80,7 @@
                          data-customize-website-views="website_sale.products_description"
                          data-no-preview="true"
                          data-reload="/"/>
-            <we-row string="Left Panel" class="o_we_full_row">
+            <we-row id="left-panel" string="Left Panel" class="o_we_full_row">
```
Comment (Williambraecky) [inline addons/website_sale/views/snippets/snippets.xml:83]: Should be more specific since ids are supposed to be unique, this is too generic, not sure whether `-` or `_` is better fit 
```suggestion
            <we-row id="o_wsale_grid_left_panel" string="Left Panel" class="o_we_full_row">
```

**Review by Williambraecky — CHANGES_REQUESTED** [review]
Late night review, looking pretty good now :)

File: addons/website/views/snippets/snippets.xml [hunk 2]
```diff
@@ -565,7 +566,7 @@
         <!-- /!\ drop-in here is partly duplicated for s_popup (see dedicated options) -->
         <!-- TODO should be improved -->
 
-    <t t-set="so_content_addition_selector" t-translation="off">blockquote, .s_card:not(.s_timeline_card), .s_alert, .o_facebook_page, .s_share, .s_social_media, .s_rating, .s_hr, .s_google_map, .s_map, .s_countdown, .s_chart, .s_text_highlight, .s_progress_bar, .s_badge, .s_embed_code, .s_donation, .s_add_to_cart</t>
+    <t t-set="so_content_addition_selector" t-translation="off">blockquote, .s_card:not(.s_timeline_card), .s_alert, .o_facebook_page, .s_share, .s_social_media, .s_rating, .s_hr, .s_google_map, .s_map, .s_countdown, .s_chart, .s_text_highlight, .s_progress_bar, .s_badge, .s_embed_code, .s_donation, .s_add_to_cart, .s_rental_search</t>
```
Comment (rdeodoo) [inline addons/website/views/snippets/snippets.xml:569]: This should be part of the `website_sale_renting` module
```xml
    <xpath expr="//t[@t-set='so_content_addition_selector']" position="inside">, .s_rental_search</xpath>
```

**Review by rdeodoo — CHANGES_REQUESTED** [review]
Quick one about the website change :+1:

File: addons/website/views/snippets/snippets.xml (hunk 2 above)
Comment (Williambraecky) [reply addons/website/views/snippets/snippets.xml:569]: Indeed thanks!

File: addons/website_sale/views/snippets/snippets.xml [hunk 3]
```diff
@@ -80,10 +80,10 @@
                          data-customize-website-views="website_sale.products_description"
                          data-no-preview="true"
                          data-reload="/"/>
-            <we-row string="Categories" data-variable="filmstrip">
+            <we-row id="o_wsale_grid_left_panel" string="Categories" data-variable="filmstrip">
                 <we-button string="Left"
-                           data-customize-website-views="website_sale.products_categories"
                            data-name="categories_opt"
+                           data-customize-website-views="website_sale.products_categories"
```
Comment (Williambraecky) [inline addons/website_sale/views/snippets/snippets.xml:86]: Revert this change

**Review by Williambraecky — APPROVED** [review]
Thanks for your work.
More information on the enterprise PR :)

robodoo r+
